setup.cfg

```
@@ -43,9 +40,9 @@ classifiers =
 include_package_data = True
 zip_safe = False
 packages = sphinx_rtd_theme
-python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*
+python_requires = >=3.8
```

Same feedback here as above, this should be

```suggestion
python_requires = >=3.6
```